daemon/emer-machine-id-provider.c

```diff
@@ -165,10 +166,14 @@ emer_machine_id_provider_class_init (EmerMachineIdProviderClass *klass)
                                      emer_machine_id_provider_props);
 }
 
-// Mandatory empty function.
+/* Explicitly set id_is_valid to false */
 static void
 emer_machine_id_provider_init (EmerMachineIdProvider *self)
 {
+  EmerMachineIdProviderPrivate *priv =
+    emer_machine_id_provider_get_instance_private (self);
+
+  priv->id_is_valid = FALSE;
```

Comment:
I don't really think this is needed – glib guarantees that the priv structure will be 0ed, and the other two fields don't get explicitly initialised here – but ok.

Reply:
Strictly speaking, that's true, though my preference is to be explicit about these kinds of things.